Upstatement/skela-wp-theme
# Delete php header and footer

## Changes

Removes the `header.php` and `footer.php` files from the directory root. Since we're rendering the page templates with Timber and twig, these two files aren't ever being referenced (right???).

## How To Test

Pull down this branch, spin up a WordPress site, and notice that everything works as normal with these two files gone.
